documentation/collection/integration/oauth/providers/auth0.md

```diff
@@ -0,0 +1,76 @@
+---
+_order: 0
+title: "Auth0"
+---
+
+OAuth integration for Auth0. Refer to [Auth0 OAuth documentation](https://auth0.com/docs/get-started/authentication-and-authorization-flow/add-login-auth-code-flow) for getting the required credentials. Provider id is `auth0`.
+
+```ts
+import { auth0 } from "@lucia-auth/oauth/providers";
+```
+
+### Initialization
+
+```ts
+import { auth0 } from "@lucia-auth/oauth/providers";
+import { auth } from "./lucia.js";
+
+const auth0Auth = auth0(auth, config);
+```
+
+```ts
+const auth0: (
+	auth: Auth,
+	config: {
+		appDomain: string;
+		clientId: string;
+		clientSecret: string;
+		redirectUri: string;
+		scope?: string[];
+		connection?: string;
+		organization?: string;
+		invitation?: string;
+		loginHint?: string;
+	}
+) => OAuthProvider;
+```
+
+#### Parameter
```

Comment:
Can you add the newly added options (`connection`, `organization`, etc) to the table?

Reply:
Always something i miss.
Added with latest commit!